install-project.sh

```diff
@@ -143,5 +150,10 @@ else
     mvn clean install -am -pl $module -DskipTests ${MVNOPTIONS} |& tee mvn-install.log
 fi
 
+cd ~/
```

Comment:
We should check if the zip already exists or not. If it does, we need not zip and copy the zip file again.